app/src/main/java/com/example/calculator/mvp/contract/CalculatorContract.java

```diff
@@ -1,5 +1,7 @@
 package com.example.calculator.mvp.contract;
 
+import com.example.calculator.utils.ErrorEnum;
+
 public interface CalculatorContract {
```

Comment:
Do not abuse with the empty lines between methods pls

Reply:
solved!